nx/lib/nx/constants.ex

```diff
@@ -333,6 +333,144 @@ defmodule Nx.Constants do
     from_binary(Nx.Type.smallest_positive_normal_binary(type), type, opts)
   end
 
+  @doc ~S"""
+  Returns $\pi$ in f32.
+  """
+  def pi, do: pi({:f, 32}, [])
```

Comment:
Why isn't type an option?

Reply:
Also, you could declare the function as `def pi(type \\ :f32, opts \\ [])` to avoid having a separate clause, and just document the default type as `{:f, 32}`.

Although it being optional might indicate it should be an option instead